runtime/lua/vim/highlight.lua

```diff
@@ -28,16 +28,30 @@ end
 --- use from init.vim via
 ---   au TextYankPost * lua require'vim.highlight'.on_yank()
 --- customize highlight group and timeout via
----   au TextYankPost * lua require'vim.highlight'.on_yank("IncSearch", 500)
+---   au TextYankPost * lua require'vim.highlight'.on_yank({higroup="IncSearch", timeout=250})
```

Comment:
I think the "pretty" way to do this is to drop the `()` and just use `on_yank { higroup = "IncSearch", timeout = 250 }`.

That is just personal preference though. I like this for all "keyword argument" style lua functions.